gc/base/GCExtensionsBase.hpp

```diff
@@ -265,6 +266,7 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	MM_SublistPool rememberedSet;
 	uintptr_t oldHeapSizeOnLastGlobalGC;
 	uintptr_t freeOldHeapSizeOnLastGlobalGC;
+	float concurrentKickoffTenuringHeadroom; /**< percentage of free memory remaining in heap, which will trigger the start of concurrent GC cycle */
```

Comment:
please state: 1) tenure heap  and 2) it's headroom on top of free memory already calculated by kickoff logic.